scripts/addModulePackageScope.js

```diff
@@ -0,0 +1,6 @@
+const fs = require('fs');
```

Comment:
I've not done this in shell to support Windows contributors.

Reply:
Not on Windows, but I really appreciate this consideration. It helps reduce the barrier to entry for contributing.